Google Cloud/Google VPC Flow Logs/ingest/parser.yml

```
@@ -0,0 +1,92 @@
+name: google_vpc_flow_logs
+pipeline:
+  - name: json_event
+    external:
+      name: json.parse-json
+      properties:
+        input_field: "{{original.message}}"
+  - name: parse_jsonPayload
+  - name: parse_labels
+  - name: google_vpc_flow_logs
+  - name: parse_connection
+  - name: parse_src_location
+  - name: parse_dest_pvc
+  - name: parse_dest_instance
+  - name: dest_gke_details
+
+stages:
+  parse_jsonPayload:
+    actions:
+      - set:
+          network.bytes: "{{json_event.message.jsonPayload.bytes_sent}}"
+          google_vpc_flow_logs.jsonPayload.end_time: "{{json_event.message.jsonPayload.end_time}}"
```

This information may be brought by `event.end`
```suggestion
          event.end: "{{json_event.message.jsonPayload.end_time}}"
```